README.md

```diff
@@ -1 +1,28 @@
-⚠️ UNDER ACTIVE DEVELOPMENT
\ No newline at end of file
+# Temporal Ruby SDK Samples
+
+This is the set of Ruby samples for the [Ruby SDK](https://github.com/temporalio/sdk-ruby).
+
+⚠️ UNDER ACTIVE DEVELOPMENT
+
+The Ruby SDK is under active development and has not released a stable version yet. APIs may change in incompatible ways
+until the SDK is marked stable.
+
+## Usage
+
+Prerequisites:
+
+* Ruby 3.1+
```

Comment:
Don't you need Go as well to run the only sample right now?

Reply:
Ah, yes, I will update the sample's README

Reply:
Actually, I saw in the README I have `with [Go](https://go.dev/) installed` which I think is good enough (did similar at https://github.com/temporalio/samples-dotnet/tree/main/src/ActivityWorker)